cmd/medius/docs/docs.go

```diff
@@ -118,9 +142,12 @@ func createDescription(artifact api.Artifact, registry string) (string, error) {
 	}
 
 	data := &docs.TemplateData{
-		Name:        metadata.Name,
-		Description: metadata.Description,
-		Example:     string(example),
+		Name:         metadata.Name,
+		Description:  metadata.Description,
+		Example:      string(example),
+		Image:        image,
+		Instancetype: metadata.EnvVariables[pkgcommon.DefaultInstancetypeEnv],
+		Preference:   metadata.EnvVariables[pkgcommon.DefaultPreferenceEnv],
```

Comment:
This may be empty, what would happen in such case?

Reply:
The template will omit empty variables, see `description.tpl`.

Reply:
All right!